docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -1,49 +1,63 @@
+. Use one of these commands to run the playbook and follow the instructions printed in the terminal:
+//+
+//Refer to https://github.com/aerogear/aerogear-digger-installer/blob/master/sample-build-playbook.yml[Digger Playbook^] to see the roles executed during installation and their associated tags.
 
-** Command to install digger without a macOS server
+** Command to install Digger with macOS server
++
+WARNING: Make sure your macOS Server is prepared for installation. See link:macos-prereqs[macOS Prerequisities].
 +
 ----
-ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=osx-provision
+ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml
 ----
-
-** Command to install digger with a macOS server
++
+** Command to install Digger without a macOS server
 +
 ----
-ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml
+ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=provision-osx
 ----
+WARNING: As of Jan 4th 2018, only `provision-osx` tag is working with with `--skip-tags`.
 
-. Carry out manual steps during the installation
 +
-There are three points during the installation where there are manual steps required.
+//** Roles can be run individually or in batch with flag //--tags=<role_tag_1>,...,<role_tag_x>
+//+
+//----
+//ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml //--tags=provision-osx,nagios
+//----
+//** To skip default roles, similarly to --skip-tags=provision-osx e.g. //--skip-tags=<role_tag_1>,...,<role_tag_x>
+//+
+//----
+//ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml //--skip-tags=provision-osx,nagios,java
+//----
+. Perform nonautomated tasks manually:
 +
-.. Accepting the Android SDK Licence
-.. Copy the server's (where ansible task are running) public ssh key to Jenkins
-.. Accept the Oracle Licence when installing Java on macOS server
+.. Accept Android SDK licence
+- During initial Android SDK installation
+- During sync of Android SDK versions
+.. If you are provisioning a macOS Server: 
+- Copy server's public SSH key(server that is running Ansible) into Jenkins Slave running on macOS server
+- Accept the Oracle License when installing Java on macOS server
 +
-Both of these steps are required for the installation to complete successfully. At each step the Ansible Installer will pause and prompt with the required actions.
 
-During the installation the Ansible playbook will perform the following tasks:
 
-* Verify and Install (if required) the minimum Java version
-* Create a project in OpenShift
-* Install Jenkins
-* Install Android SDK to a Persistent Volume
-* Configure Jenkins
-* Setup macOS Server(s) (optional depending on command used)
-* Install Nagios
-* Print url and credentials for Jenkins and Nagios
+.Ansible playbook will perform the following tasks:
+
+* Check if the required Java version is installed, install/upgrade if not,
```

Comment:
@laurafitzgerald could you please review this section?

Reply:
@pmacko1 yep this matches with the roles specified in the [playbook](https://github.com/aerogear/aerogear-digger-installer/blob/master/sample-build-playbook.yml)

Reply:
resolved